Tasks/AzureCLIV2/task.json

```diff
@@ -0,0 +1,200 @@
+{
+    "id": "46E4BE58-730B-4389-8A2F-EA10B3E5E815",
+    "name": "AzureCLI",
+    "friendlyName": "Azure CLI",
+    "description": "Run Azure CLI commands against an Azure subscription in a Shell/PowerShell script when running on Linux agent or Batch/PowerShell script when running on Windows agent.",
+    "author": "Microsoft Corporation",
+    "helpUrl": "https://docs.microsoft.com/azure/devops/pipelines/tasks/deploy/azure-cli",
+    "helpMarkDown": "[Learn more about this task](http://github.com/microsoft/azure-pipelines-tasks/blob/master/Tasks/AzureCLIV2/Readme.md) or [see the Azure CLI documentation](https://docs.microsoft.com/cli/azure/)",
+    "releaseNotes": "What's new in Version 2.0:\n- Support for PowerShell script \n- Works with cross-platform agents (Linux, macOS, or Windows)\n- For working with Azure CLI 1.0 (node.js-based), switch to task version 0.0\n- Limitations:\n - No support for Azure Classic subscriptions. Azure CLI 2.0 supports only Azure Resource Manager (ARM) subscriptions.",
```

Comment:
Do you want to add "PowerShell Core" as well?
Apart from first line, all others are applicable only for previous version. Do you want to keep them?